Tests/FetchingLogictests/UtilsTests.swift

```diff
@@ -84,20 +84,95 @@ import XCTest
     }
 }
 
-class UtilsTests: XCTestCase {
+@objcMembers class MPNativeAdRequest: NSObject {
+    
+    var name: String!
+    private(set) var p_customTargeting: MPNativeAdRequestTargeting
```

Comment:
`p_` ?

Reply:
p_ is already used in the existing test classes hence used the same here